Makefile

```diff
@@ -51,8 +51,7 @@ check-blog: ## Checks that all blog articles are signed by their authors
 CHECK_URLS=false
 export CHECK_URLS
 generate: ## Generates files across the repo using the data in /.meta
-	@bundle install --gemfile=scripts/Gemfile --quiet
-	@scripts/generate.rb
+	scripts/docker-run.sh checker scripts/generate.rb
```

Comment:
Might it make sense to only change Dockerfiles in this PR, and implement the user-facing parts of https://github.com/timberio/vector/issues/1613 in a separate PR?

Reply:
Sure, I can do that.